packages/polling-controller/src/StaticIntervalPollingController.ts

```diff
@@ -89,3 +90,10 @@ export const StaticIntervalPollingController = <PollingInput extends Json>() =>
   StaticIntervalPollingControllerMixin<typeof BaseController, PollingInput>(
     BaseController,
   );
+
+export const StaticIntervalPollingControllerNext = <
+  PollingInput extends Json,
+>() =>
+  StaticIntervalPollingControllerMixin<typeof BaseControllerNext, PollingInput>(
+    BaseControllerNext,
+  );
```

Comment:
This has been added to untangle `gas-fee-controller` from other polling controllers, and is being removed in https://github.com/MetaMask/core/pull/6444